app/uk/gov/hmrc/vatsubscription/models/PPOBAddress.scala

```diff
@@ -21,13 +21,13 @@ import play.api.libs.json._
 import uk.gov.hmrc.vatsubscription.utils.JsonReadUtil
 
 
-case class PPOBAddress(line1: Option[String],
+case class PPOBAddress(line1: String,
                        line2: Option[String],
                        line3: Option[String],
                        line4: Option[String],
                        line5: Option[String],
-                       postCode: Option[String],
-                       countryCode: Option[String])
+                       postCode: String,
```

Comment:
If the address is foreign then the postcode is not mandatory; so would be best to revert that back to optional.